sage-ios/SAGE/SAGE/BaseOperation.swift

```diff
@@ -24,6 +24,10 @@ class BaseOperation {
     // This should be added to every params with any request while logged in
     class func addAuthParams(manager: AFHTTPRequestOperationManager) {
```

Comment:
can you actually change this to private too? we will only need this in this class